<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmreports</class>
 <widget class="QWidget" name="frmreports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Choose one report from the list:</string>
       </property>
       <property name="buddy">
        <cstring>listView</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView">
       <property name="toolTip">
        <string>Available reports</string>
       </property>
       <property name="statusTip">
        <string>Available reports on the report directory</string>
       </property>
       <property name="whatsThis">
        <string>Lists the available reports on the report directory</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Browse to load a report:</string>
       </property>
       <property name="buddy">
        <cstring>lineOpen</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineOpen"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushOpen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Open report</string>
         </property>
         <property name="statusTip">
          <string>Browse files to load a report from non-standard location</string>
         </property>
         <property name="whatsThis">
          <string>Browse files to load a report from non-standard location</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/folder.png</normaloff>:/app_new/folder.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Report Properties</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Icon:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbAuthor">
          <property name="toolTip">
           <string>Report Author</string>
          </property>
          <property name="statusTip">
           <string>Report Author</string>
          </property>
          <property name="whatsThis">
           <string>Report Author</string>
          </property>
          <property name="text">
           <string>-&gt; TYPE SOMETHING HERE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbPixmap">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Report Icon</string>
          </property>
          <property name="statusTip">
           <string>Report Icon</string>
          </property>
          <property name="whatsThis">
           <string>Report Icon</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="app_new.qrc">:/app_new/kpresenter.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Description:</string>
          </property>
          <property name="buddy">
           <cstring>textDescription</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextBrowser" name="textDescription">
          <property name="toolTip">
           <string>Report description</string>
          </property>
          <property name="statusTip">
           <string>Report description</string>
          </property>
          <property name="whatsThis">
           <string>Report description</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushClose">
         <property name="toolTip">
          <string>Close</string>
         </property>
         <property name="statusTip">
          <string>Close</string>
         </property>
         <property name="whatsThis">
          <string>Close</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>textDescription</tabstop>
  <tabstop>lineOpen</tabstop>
  <tabstop>pushOpen</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections/>
</ui>
